scripts/migrations/tests/1_deploy-contracts.js

```diff
@@ -0,0 +1,59 @@
+const AccessControlConfig = artifacts.require('./8.17/stablecoin-core/config/AccessControlConfig.sol');
```

Comment:
Scripts here eventually need to be same as deployment sripts, however, due to differences it would take quite a lot of time to adjust. Not so much urgent, therefore, would postpone it till urgent matters(setPrice, fairLaunch replacement, feeStructure) are taken care of.